src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,173 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    public MyLinkedList() {
+        this.first = null;
+        this.last = null;
```

Comment:
[where should you use this](stackoverflow.com/a/2411283)
does initializing with null change anything? What is the default value of uninitialized object?